corehq/messaging/scheduling/models/content.py

```diff
@@ -155,7 +155,6 @@ def send(self, recipient, logged_event, phone_entry=None):
         email.save()
 
         email_usage.update_count()
-        logged_subevent.completed()
```

Comment:
What will show up in the report between this point and when the email is marked delivered?

Reply:
The `logged_subevent` gets created with `STATUS_IN_PROGRESS`, so that's what will show up until some other status is set. 

https://github.com/dimagi/commcare-hq/blob/65f2486c68ab317c15b4856a47a0483e40edcac4/corehq/apps/sms/models.py#L1310-L1321